frontend/src/components/Header.js

```diff
@@ -2,13 +2,15 @@ import {AppBar,  Toolbar, Typography} from "@mui/material";
 import AccountBalanceIcon from '@mui/icons-material/AccountBalance';
 
 export default function Header() {
+
+
     return (
         <header>
             <meta name="viewport" content="initial-scale=1, width=device-width"/>
             <AppBar position="relative">
                 <Toolbar>
                     <AccountBalanceIcon fontSize="large"/>
-                    <Typography variant="h4">My Depot</Typography>
+                    <Typography variant="h4"> &nbsp;My Depot </Typography>
```

Comment:
```suggestion
                    <Typography variant="h4" sx={{pl:1}}> My Depot </Typography>
```
better to use padding here instead of a space in my opinion